ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -230,13 +230,14 @@ export default function BuildQuote ({
         <div
           className="build-quote__swap-arrows-row"
         >
-          <div
+          <button
             className="build-quote__swap-arrows"
             onClick={() => {
               onToSelect(selectedFromToken)
               onFromSelect(selectedToToken)
             }}
-          />
+          ><img src="/images/icons/swap2.svg" alt="" width="12" height="16" />
```

Comment:
@danjm Can I simply add keys and values to `app/_locales/en/messages.json`?  Then localizers will add them to other files?

Also, what label do we want to give this?  I'm having trouble thinking of the proper phrase.

Reply:
>  Can I simply add keys and values to app/_locales/en/messages.json? Then localizers will add them to other files?
Yes!

> Also, what label do we want to give this? I'm having trouble thinking of the proper phrase.
Something like "Switch from and to tokens"? Or "Switch source and destination tokens"?